src/Computation.hs

```diff
@@ -224,6 +227,56 @@ regionX' :: CompFn a
 regionX' [] [LN' lineLeft, LN' lineRight] = TNum $ regionX lineLeft lineRight
 regionX' v o = error' "regionX" v o
 
+-- returns the middle of the square as a starting position
+midSquare' :: CompFn a
+midSquare' [] [S' b] = TPt((xs' b),(ys' b))
```

Comment:
maintain spacing everywhere and avoid unnecessary parens: TPt (xs' b, ys' b)

Reply:
Again, we should all use Hlint 😄